web/themes/custom/vote_gov/templates/partial/register/registration-type--not-needed.html.twig

```diff
@@ -0,0 +1,2 @@
+<h2>{{ 'How to register to vote in @state_name' | t({ '@state_name': label | render }) }}</h2>
+<div>{{ 'Voter registration is not required in @state_name. Learn more about voting on @state_name\'s election website.' | t({ '@state_name': label | render }) }}</div>
```

Comment:
replace `<div>` wrapper with `<p>`
add the missing link to state election website